recipes/octo-logger-cpp/all/conanfile.py

```diff
@@ -61,13 +75,21 @@ def source(self):
 
     def requirements(self):
         self.requires("fmt/9.0.0")
+        if self.options.with_aws and self._aws_supported:
+            self.requires("nlohmann_json/3.11.2")
+            self.requires("aws-sdk-cpp/1.9.234")
 
     def build_requirements(self):
         self.build_requires("cmake/3.24.0")
 
     def build(self):
         cmake = CMake(self)
-        cmake.configure()
+        variables = {}
+        if self._aws_supported:
+            variables.update({
+                "WITH_AWS": self.options.with_aws
+            })
+        cmake.configure(variables=variables)
```

Comment:
```suggestion
        cmake.configure()
```
It should be done in CMakeToolchain

Reply:
gotcha, fixed